scripts/prepare_sector_network.py

```diff
@@ -752,6 +753,10 @@ def add_aviation(n, cost):
     )
 
 
+def new_func():

```

Comment:
What is the purpose of this function?

Reply:
As you can see, nothing at all 😆 
My bad. I probably pressed something in vscode that created it automatically.